debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -110,6 +112,8 @@ else if (70 <= year && year <= 99) {
         return temporal;
     }
 
+    private final ParsingErrorHandler parsingErrorHandler;
```

Comment:
Pushed a commit to make this final (let's always strive for that), which was a good ocassion to clean-up the constructor mess in this class a little bit.